src/uu/ls/ls.md

```diff
@@ -1,8 +1,6 @@
 # ls
 
-## Usage
-
-```
+```sh
```

Comment:
You couldn't have known this, but `sh` might break our macros. Have you checked that the output in `--help` and `uudoc` are still correct with this?

Reply:
Hi @tertsdiepraam, I checked --help and it behaves as before. As per code documentation it should be allowed -> https://github.com/uutils/coreutils/blob/b9bca794cca17fa3ee3b63a2a81e22c253b47821/src/uucore_procs/src/lib.rs#L94

Reply:
You're right that it does indeed work! Thanks for checking!

(The code fence you're linking to is actually irrelevant because those code fences are not part of the syntax that we parse but a markdown code block parsed by rustdoc to output this: https://docs.rs/uucore/0.0.17/uucore/macro.help_section.html, also it's on another function.)

Reply:
As we should be consistent, 
if we use ```sh
we should use it elsewhere too.

@tertsdiepraam what do you think we should do?

Reply:
Good point. I'm leaning towards not adding `sh`, because what we're writing is not strictly `sh` syntax.